src/components/dungeonView.html

```diff
@@ -116,5 +100,61 @@ <h4><strong data-bind="text: DungeonRunner.timeLeftSeconds() + 's'"></strong></h
             <button class="btn btn-warning dungeon-button">Leave Dungeon</button>
         </div>
         <!-- /ko -->
+        <h2 class="pageItemFooter" style="display: block;">
+          <table width="100%">
+              <tr>
+                <td width="50%">
+                  <knockout data-bind="text: DungeonRunner.dungeon.name">Dungeon name</knockout>
+                  <!--If all Pokémon in the dungeon are caught-->
+                  <knockout data-bind="if: (!DungeonRunner.dungeonCompleted(DungeonRunner.dungeon, true) && DungeonRunner.dungeonCompleted(DungeonRunner.dungeon, false))">
+                      <img title="You have captured all Pokémon in this dungeon!" class="pokeball-smallest"
+                           src="assets/images/pokeball/Pokeball.svg"/>
+                  </knockout>
+
+                  <!--If all Pokémon in the dungeon are caught shiny-->
+                  <knockout data-bind="if: DungeonRunner.dungeonCompleted(DungeonRunner.dungeon, true)">
+                      <img title="You have captured all Pokémon shiny in this dungeon!"
+                           class="pokeball-smallest"
+                           src="assets/images/pokeball/Pokeball-shiny.svg"/>
+                  </knockout>
+
+                  <!--Pokérus image-->
+                  <knockout data-bind="if: RouteHelper.minPokerusCheck(DungeonRunner.dungeon.allAvailablePokemon())">
+                      <img data-bind="attr:{ src: 'assets/images/breeding/pokerus/' + GameConstants.Pokerus[RouteHelper.minPokerus(DungeonRunner.dungeon.allAvailablePokemon())] + '.png'}"
+                    src=""/>
+                  </knockout>
+                </td>
+                <td width="50%">
+                  <knockout data-bind="using: App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(DungeonRunner.dungeon.name)]()"</knockout>
+                  <knockout data-bind="text: $data.toLocaleString('en-US')">0</knockout>
+                  &nbsp;Clears
+                </td>
+              </tr>
+              <tr>
+                <td width="50%">
+                    <span style="display: inline;">
+                        Pokémon Attack:
+                        <span data-bind="
```

Comment:
Can we throw this span into a template? It's used a lot, and is quite big

Reply:
Yes, but I can't get the tooltip to work. Anybody that understands HTML a bit more than I feel free to try and fix up pokemonAttackTemplate.html